docs/guides/serverless-first-program.ipynb

```diff
@@ -329,7 +327,121 @@
     }
    ],
    "source": [
-    "serverless.list()[:3]  # Limit to first three functions"
+    "# Get program from serverless.list() that matches the title of the one we uploaded\n",
+    "next(\n",
+    "    program for program in serverless.list()\n",
+    "    if program.title == \"transpile_remote_serverless\"\n",
+    ")"
```

Comment:
I changed this to better fit the prose. We're trying to show the program uploaded successfully, but the output actually only shows other programs.